.travis/custom_release.sh

```diff
@@ -21,4 +21,10 @@ then
         rm -rf ./dist/.git
         .travis/release.sh "${env}-stable"
     done
+fi
+
+if [[ "${TRAVIS_BRANCH}" = "prod-beta" || "${TRAVIS_BRANCH}" = "prod-stable" ]]; then
```

Comment:
Hello @karelhala, I took the reference from [here](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/deployments/setup/DOC.md#custom-releases)

For rest of branches, it runs `rm -rf ./dist/.git` command but for `prod` branches it says to run `rm -rf ./build/.git`. Could you help me in understanding why there is difference in directory path?

Reply:
I don't think it really matters, you don't have to remove either of those. If you look at your code the `.git` folder is not located in either of those.

Reply:
Thank you @karelhala!
Yes, there is no `.git` folder.
IIRC, last time I have faced an issue where build files were not copied to build repository & job failed with error.
So keeping the changes as it is.